src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java

```diff
@@ -95,6 +95,20 @@ public ResponseEntity<Dispute> getDispute(@PathVariable Long id, Principal princ
 		}
 		return new ResponseEntity<Dispute>(disputeService.getDisputeById(id), HttpStatus.OK);
 	}
+	
+	/**
+	 * GET endpoint that retrieves the detail of a specific dispute without assigning it
+	 *
+	 * @param id
+	 * @param principal the logged-in user
+	 * @return {@link Dispute}
+	 */
+	@GetMapping("/dispute/noassign/{id}")
```

Comment:
Is this better than adding a parameter to the existing? ie. `/dispute/{id}?noassign=true`

Reply:
you are right i reconfigured this but this controller method was called widely so all the calls had to be changed as well, np it is done